lib/src/svg/colors.dart

```diff
@@ -32,20 +32,19 @@ Color parseColor(String colorString) {
   }
 
   // handle rgb() colors e.g. rgb(255, 255, 255)
-  if (colorString.startsWith('rgb')) {
-    final List<int> rgb = colorString
+  if (colorString.toLowerCase().startsWith('rgb')) {
+    final List<int> rgb = <int>[];
```

Comment:
I'd rather avoid using a dynamically sized array when we know the expected length.  This could be `new List<int>(4);` and then the loop below would have to be modified.  Otherwise, you could just add `rawColor = rawColor.trim()` into the function passed into `map` in the original code.